go/test/endtoend/vtgate/queries/misc/misc_test.go

```diff
@@ -301,6 +301,6 @@ func TestLeftJoinUsingUnsharded(t *testing.T) {
 	mcmp, closer := start(t)
 	defer closer()
 
-	utils.Exec(t, mcmp.VtConn, "insert into uks.unsharded(id1) values (1),(2),(3),(4),(5)")
-	utils.Exec(t, mcmp.VtConn, "select * from uks.unsharded as A left join uks.unsharded as B using(id1)")
+	utils.Exec(t, mcmp.VtConn, "insert /*vt+ QUERY_TIMEOUT_MS=1000 */ into uks.unsharded(id1) values (1),(2),(3),(4),(5)")
```

Comment:
Apparently the test setup defaults this to 100ms and test was failing after that time.

Reply:
Correct, the query timeout is set to 100ms in the `TestMain`

Reply:
It's probably the select that is slow. the Insert is expected to be fast on the kind of small test data we use for tests. It's fine to leave the timeout for both queries, it won't hurt anything.